modules/domain-mongo/sources/MongoAggregateStore.kt

```diff
@@ -35,6 +36,13 @@ private class MongoAggregateStore(
 		}
 	}
 
+	override suspend fun lastEventTimestampOrNull(): Timestamp? =
+		collection.find()
+			.sort(descending(Fields.id))
+			.limit(1)
```

Comment:
There's no need to set a limit when using `firstOrNull()`. It's optimized by the MongoDB Driver already.